src/js/common/sentry.js

```diff
@@ -0,0 +1,22 @@
+import Raven from 'raven-js';
+
+import environment from './helpers/environment';
+
+// url check is necessary for e2e tests
+const trackErrors = (__BUILDTYPE__ === 'production' || __BUILDTYPE__ === 'staging')
```

Comment:
I think I'd still be in favor of keeping this active on dev.vets.gov, too. Can we just hold off on the __BUILDTYPE__ check and focus only on the BASE_URL?